packages/block-library/src/terms-query/inspector-controls/display-options.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import {
+	__experimentalToolsPanelItem as ToolsPanelItem,
+	RadioControl,
+} from '@wordpress/components';
+
+const getOptions = ( displayTopLevelControl, displaySubtermsControl ) => {
+	const options = [ { label: __( 'Show all' ), value: 'all' } ];
+
+	if ( displayTopLevelControl ) {
+		options.push( {
+			label: __( 'Show only top level terms' ),
+			value: 'top-level',
+		} );
+	}
+
+	if ( displaySubtermsControl ) {
+		options.push( {
+			label: __( 'Show subterms only' ),
+			value: 'subterms',
+			description: __(
+				'Display subterms of the current term. E.g. subcategories of current category.'
+			),
+		} );
+	}
+
+	return options;
+};
+
+const allTermsQuery = {
+	include: [],
+	exclude: [],
+	parent: false,
+};
+
+const topLevelTermsQuery = {
+	include: [],
+	exclude: [],
+	parent: 0,
+	hierarchical: false,
+};
+
+const getQueryAttributes = ( value ) => {
+	// For subterms we fetch all terms and then filter them since we're building the tree
+	// ourselves in Term Template.
+	if ( value === 'all' || value === 'subterms' ) {
+		return allTermsQuery;
+	}
+	if ( value === 'top-level' ) {
+		return topLevelTermsQuery;
+	}
+};
```

Comment:
Do you think we could simplify this slightly by switching the order around and defaulting to returning `allTermsQuery`?

```suggestion
const getQueryAttributes = ( value ) => {
	if ( value === 'top-level' ) {
		return topLevelTermsQuery;
	}
	// For 'all' and 'subterms', we fetch all terms and then filter them as the tree is built in Term Template.
	return allTermsQuery;
};
```

Reply:
Good idea, done in https://github.com/WordPress/gutenberg/pull/71633/commits/4020cb37f8b0df5dea80956cd3ea35937501291e